### PR TITLE
Python documentation

### PR DESCRIPTION
**Pull request by janden**

### CODE REVIEW

File: docs/python.rst [hunk 1]
```diff
@@ -1,23 +1,125 @@
 Python interface
 ================
 
-*** TO DO ***
+Quick-start examples
+--------------------
 
-*** here's the old stuff...
+These Python interfaces are written by Libin Lu, with help from David Stein, Alex Barnett, and Joakim Andén.
```
Comment (janden) [inline docs/python.rst:7]: Remove this.

**Comment by janden** [comment]
Update with new interface and add note about single precision.

**Comment by janden** [comment]
@ahbarnett @lu1and10 this is done and ready for review

**Comment by ahbarnett** [comment]
Looks great!

There's several (minor) qu's:

* you're going with finufftpy as the module name not finufft - is that for bkw compat?
* could the first example have a out=f variant, maybe just a mention?
*  for 2d I suggest rectangular (N2 .neq. N1) example, like I did in matlab, since it teaches the ordering of modes (which as we know can be confusing!).
* for stacking you say it's stacked on the "last" dim - is that what python people call K in a shape (K,M)? You mean first dim?
* finufftpy.Plan    this is Caps and not makeplan due to some py OO constraint, right?
* docstrings - don't forget to update python_tests to python/tests/

Thanks! Alex

**Comment by ahbarnett** [comment]
I thought edits would be better done in the main gi_v2 branch since they have to be done to examples and docs at the same time. Plus I can't edit your PR.  Best, Alex

**Comment by janden** [comment]
> I thought edits would be better done in the main gi_v2 branch since they have to be done to examples and docs at the same time.

Sounds good. Your comments all make sense and I will address them tomorrow.

> Plus I can't edit your PR.

Sorry about that. There's a box I forgot to check. Will make sure to do that next time.

**Pull request merged**